src/crypto/aead.rs

```diff
@@ -189,7 +191,11 @@ mod ring {
                     let secret = KeyMaterial(&test_vector.key_material)
                         .expand_as_secret(&cipher_suite)
                         .unwrap();
-                    let nonce = FrameNonceSequence::new(test_vector.frame_count, &secret.salt);
+                    let nonce = FrameNonceSequence::new(
+                        &FrameCount::from(test_vector.frame_count),
```

Comment:
this is a bit weird with the temporaries. Do we maybe just want to do this with an `.into()` ?

Reply:
I will check that